From e43c7f22aad716a134ac6dab1d767e9de5b94038 Mon Sep 17 00:00:00 2001
From: Zorro Liu <lyx@rock-chips.com>
Date: Mon, 8 Oct 2018 11:22:25 +0800
Subject: [PATCH] arm64: dts: rockchip: update configs for
 rk3399pro-npu-evb-v10 board

1.add npu node
2.fix fiq interrupt num
3.add usb otg
...

Change-Id: I9c03a86f673e6dd98fadc89473ff8563769c38b8
Signed-off-by: Zorro Liu <lyx@rock-chips.com>
---
 .../dts/rockchip/rk3399pro-npu-evb-v10.dts    | 25 +++++++++++++++++--
 1 file changed, 23 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts
index 94939e2edaba..03f97f974a7e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts
@@ -36,7 +36,7 @@
 		/* If enable uart uses irq instead of fiq */
 		rockchip,irq-mode-enable = <0>;
 		rockchip,baudrate = <1500000>;  /* Only 115200 and 1500000 */
-		interrupts = <GIC_SPI 274 IRQ_TYPE_LEVEL_LOW>;
+		interrupts = <GIC_SPI 242 IRQ_TYPE_LEVEL_HIGH>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&uart2m0_xfer>;
 		status = "okay";
@@ -83,7 +83,7 @@
 		regulator-min-microvolt = <850000>;
 		regulator-max-microvolt = <850000>;
 		regulator-ramp-delay = <1000>;
-		fcs,suspend-voltage-selector = <1>;
+		fcs,suspend-voltage-selector = <0>;
 		regulator-boot-on;
 		regulator-state-mem {
 			regulator-off-in-suspend;
@@ -91,6 +91,27 @@
 	};
 };
 
+&npu {
+	npu-supply = <&vdd_npu>;
+	status = "okay";
+};
+
+&u2phy {
+	status = "okay";
+};
+
+&u2phy_otg {
+	status = "okay";
+};
+
+&usbdrd3 {
+	status = "okay";
+};
+
+&usbdrd_dwc3 {
+	status = "okay";
+};
+
 &pinctrl {
 	vsel_gpio: vsel-gpio {
 		rockchip,pins =
-- 
2.35.3

